### PR TITLE
Feature/enhance card

### PR DESCRIPTION
**Pull request by steffilankes**

### CODE REVIEW

File: packages/components/src/components/card/card.tsx [hunk 1]
```diff
@@ -30,8 +30,19 @@ export class card {
...
       <div class={this.getCssClassMap()}>
+        <div class="card__header">
+        <h4 class="card-title">Testtitel Header</h4>
+        <h5 class="card-subtitle">Subtitel Header</h5>
+          <slot name="header" />
+        </div> 
+        <img class="card-img-top" src="http://placehold.it/400x300" alt="Card image"></img>
         <div class="card__body">
           <slot/>
+          <t-button variant="primary" class="hydrated">Click me!</t-button>
```
Comment (eeegor) [inline packages/components/src/components/card/card.tsx]: no need to embed `t-button`, it should come in a slot (probably will be injected as part of of the default slot)

File: packages/components/src/components/card/card.tsx [hunk 2]
```diff
@@ -30,8 +30,19 @@ export class card {
   render() {
     return (
       <div class={this.getCssClassMap()}>
+        <div class="card__header">
+        <h4 class="card-title">Testtitel Header</h4>
+        <h5 class="card-subtitle">Subtitel Header</h5>
+          <slot name="header" />
+        </div> 
+        <img class="card-img-top" src="http://placehold.it/400x300" alt="Card image"></img>
```
Comment (eeegor) [inline packages/components/src/components/card/card.tsx]: `class="card-img-top"` should be `class="card__img-top"`

File: packages/components/src/components/card/card.tsx [hunk 3]
```diff
@@ -30,8 +30,19 @@ export class card {
...
         <div class="card__body">
           <slot/>
+          <t-button variant="primary" class="hydrated">Click me!</t-button>
+        </div>
+        <div class="card__footer">
+        <h4 class="card-title">Testtitel Footer</h4>
```
Comment (eeegor) [inline packages/components/src/components/card/card.tsx]: No need to hardcode `<h4 class="card-title">Testtitel Footer</h4>`, will be injected by user in a slot

File: packages/components/src/components/card/card.tsx (hunk 2 above)
Comment (steffilankes) [reply packages/components/src/components/card/card.tsx]: also at `card-title ` and `card-subtitle` ?

File: packages/components/src/components/card/card.tsx (hunk 3 above)
Comment (steffilankes) [reply packages/components/src/components/card/card.tsx]: also at `header` ?

File: packages/components/src/components/card/card.tsx (hunk 2 above)
Comment (eeegor) [reply packages/components/src/components/card/card.tsx]: yes please

File: packages/components/src/components/card/card.tsx (hunk 3 above)
Comment (eeegor) [reply packages/components/src/components/card/card.tsx]: yes

File: packages/components/src/components/card/card.e2e.ts [hunk 4]
```diff
@@ -8,3 +8,4 @@ describe('t-card', () => {
     expect(element).toHaveClass('hydrated');
   });
 });
+// Test für bild, alt-text
```
Comment (eeegor) [inline packages/components/src/components/card/card.e2e.ts]: Please use English for comments ;)

File: packages/components/src/components/card/card.tsx [hunk 5]
```diff
@@ -30,9 +32,16 @@ export class card {
   render() {
     return (
       <div class={this.getCssClassMap()}>
+        <div class="card__header">
+        <slot name="header" />
```
Comment (eeegor) [inline packages/components/src/components/card/card.tsx]: Please indent `<slot name="header" />`

File: packages/components/src/components/card/card.e2e.ts (hunk 4 above)
Comment (steffilankes) [reply packages/components/src/components/card/card.e2e.ts]: it's okay. this comment is only for me :)

Comment (eeegor) [reply packages/components/src/components/card/card.e2e.ts]: If it's merged, it's for everyone

**Pull request merged**